resources/js/store_comment.js

```diff
@@ -5,7 +5,7 @@ $(document).ready(function () {
         }
     });
 
-    $('.store-comment').click(function() {
+    $('.store-comment').on('click', function () {
```

Comment:
on click thì nên chỉ định click vào cái gì chứ nhỉ
để như này thì this là `.store-comment` luôn à e? có gì hay ho hơn cách trên ko?

Reply:
e chuyển thành body onclick để lúc phần load-more append thêm dữ liệu thì mới comment đc